src/main/resources/application.properties

```diff
@@ -1,3 +1,4 @@
 spring.application.name=medical-clinic-proxy
+server.port=8085
 
-spring.cloud.openfeign.client.config.medicalClinicProxyClient.url=http://localhost:8080
\ No newline at end of file
+spring.cloud.openfeign.client.config.medicalClinicProxyClient.url=http://medical-clinic:8080
```

Comment:
tutaj powinnismy miec kofniguracje ktora zadziala klikajac run w intellij. To nie zadziala.

Reply:
zmienilem na http://localhost:8080, na ktorym lokalnie stoi medical-clinic i dziala po uruchomieniu lokalnie przez run, a nie przez dockera